### PR TITLE
fixes as proposed in the card

### PR DESCRIPTION
**Pull request by alexnb**
[Trello card](https://trello.com/c/gGyzDLNk/1147-feedback-on-docs-page)
----

If you open a PR that needs to go into a current version, you need to *cherry-pick your commit from dev over to the current version branch*. Only then will the proper builds that generate html/pdf be run. But beware: Docs will be generated but not published automatically!

- [x] N/A - or - I have added the appropriate "cherry-pick-to" labels to this PR so I don't forget to do this later!

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

File: modules/ROOT/pages/introduction/overview.adoc [hunk 1]
```diff
@@ -10,13 +10,13 @@ Monitoring has been achievable through integration of 3rd party tooling, requiri
 None of these solutions expose natively the Neo4j context, and there has been an absence of a single place to monitor, administer and operate the customers entire Neo4j estate.
 
 Neo4j Ops Manager (abbreviated to NOM) is a tool provided to assist an administrator of Neo4j DBMS deployments and address this gap.
-It contains current and future features which allow the administrator to monitor, administer and operate these deployments within their estate.
+It provides features which allow the administrator to monitor, administer and operate these deployments within their estate.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/introduction/overview.adoc]:
```suggestion
It provides features which allow the administrator to monitor, administer, and operate these deployments within their estate.
```

File: modules/ROOT/pages/introduction/overview.adoc [hunk 2]
```diff
@@ -10,13 +10,13 @@ Monitoring has been achievable through integration of 3rd party tooling, requiri
...
 
 NOM is built on a core architecture designed to support multiple DBMS implementations, both single instance and clusters. This architecture contains metadata driven extensible frameworks for intelligent monitoring and a knowledge engine.
 With these features, is it possible for NOM to assist the Administrator with understanding not just what is going on, but why it is happening and make recommendations explaining what to do about it through explainable insight.
 On top of this architecture and framework are a set of “managers”, covering four pillars of Monitoring, Administration, Operations and Integration.
 
-Whilst NOM contains initially a small set of features, it will continue to add capabilities in three ways:
+NOM will add capabilities in three ways:
```
Comment (AlexicaWright) [inline modules/ROOT/pages/introduction/overview.adoc]:
```suggestion
NOM adds capabilities in three ways:
```

**Review by AlexicaWright — COMMENTED** [review]
Just two editorial comments.

**Pull request merged**